src/assets/styles/global.scss

```diff
@@ -1,4 +1,4 @@
-@import "./node_modules/@esri/calcite-colors/colors.scss";
+@import "./node_modules/@esri/calcite-colors/src/colors.scss";
```

Comment:
Unrelated: Maybe we should include the sass file as part of the distributable too and not ship the source.

Reply:
Yeah I noticed that too. I think they make sense, can we update them after?